src/effects/VST/VSTEffect.cpp

```diff
@@ -374,7 +374,7 @@ unsigned VSTEffectsModule::DiscoverPluginsAtPath(
          //Subsequent VSTEffect::Load may seem like overhead, but we need
          //to initialize EffectDefinitionInterface part, which includes
          //properly formatted plugin path
-         VSTEffect subeffect(wxString::Format("%s;%d", path, id), &effect);
+         VSTEffect subeffect(wxString::Format("%s;%d", path, id));
```

Comment:
In step 5.3, add:
`subeffect.Load()`
to preserve the behavior.

Reply:
Good catch! ok